bin/helpers/capabilityHelper.js

```diff
@@ -138,9 +142,22 @@ const caps = (bsConfig, zip) => {
     if (obj.parallels) logger.info(`Parallels limit specified: ${obj.parallels}`);
 
     var data = JSON.stringify(obj);
+
     resolve(data);
   })
 }
+const getAccessibilityPlatforms = (bsConfig) => {
+  const browserList = bsConfig.browsers;
+  const accessibilityPlatforms = Array(browserList.length).fill(false);
+  let rootLevelAccessibility = false;
+  if (bsConfig.run_settings.accessibility !== undefined) {
```

Comment:
Use the `isUndefined` util method.
https://github.com/browserstack/browserstack-cypress-cli/blob/45027801c24b09e55c2dfb46808b39c9a710e2c4/bin/helpers/utils.js#L594

Reply:
done